modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js

```diff
@@ -129,6 +130,12 @@ class ContributorBuilder extends React.Component {
 			'criteria-builder-section-sidebar--with-warning': showDisabledSegmentationAlert,
 		});
 
+		function handleClick(event) {
```

Comment:
Maybe a more specific name for this function would be better. Something like `handleViewMembersClick`. What do you think?

Reply:
Done! thank you!